module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -646,6 +646,21 @@ public function getHolding($bibId, array $patron = null, array $options = [])
         return $items;
     }
 
+    /**
+     * Convert a FOLIO date string to a DateTime object.
+     *
+     * @param string $str FOLIO date string
+     *
+     * @return DateTime
+     */
+    protected function getDateTimeFromString(string $str): DateTime
```

Comment:
Created a shared support method since we did the same date conversion in two places.